ocs_ci/helpers/helpers.py

```
@@ -7669,3 +7671,220 @@ def delete_cephfs_ec_pool(pool_name):
+            namespace=namespace,
+            data_dict={".dockerconfigjson": docker_config},
+            secret_type="kubernetes.io/dockerconfigjson",
+        )
+
+    # 3. Create Cluster CR
+    scale_cluster_kind = "Cluster.scale.spectrum.ibm.com"
+    scale_cluster_ocp = OCP(kind=scale_cluster_kind, namespace=namespace)
+
+    try:
+        scale_cluster_ocp.get(resource_name=cluster_name)
+        logger.info(f"Scale Cluster CR '{cluster_name}' already exists.")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)